lib/gui/modules/analytics.js

```diff
@@ -142,13 +142,15 @@ analytics.service('AnalyticsService', function($log, $window, $mixpanel, Setting
    */
   this.logEvent = (message, data) => {
     const flatStartCaseData = utils.makeFlatStartCaseObject(data);
+    const anonymizedFlatStartCaseData = utils.hideObjectAbsolutePaths(flatStartCaseData);
```

Comment:
What about `const debugData = utils.hideAbsolutePathsInObject(utils.makeFlatStartCaseObject(data));`? It loses information fromthe naming, but is not a mouthful anymore.